app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -278,9 +278,32 @@ export const ContentForm: React.FC<IContentFormProps> = ({ contentType = Content
                           );
                         }}
                       />
-                      <ContentActions init filter={(a) => a.valueType === ValueType.Boolean} />
+                      {/* section one of actions */}
+                      <Col alignSelf="stretch">
+                        <ContentActions
+                          init
```

Comment:
Trying to match mockup with the action groupings

Reply:
Probably not worth the time.  I don't think the UI was designed specifically for an ordered reason for the majority of these.  We can control the order with the table column `sortOrder` anyway.